### PR TITLE
[JEP-223] - Shorten the JEP title

### PR DESCRIPTION
**Pull request by oleg-nenashev**
CC @mikecirioli @aHenryJard @EstherAF . The current title is a bit too long

### CODE REVIEW

File: jep/223/README.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= JEP-223: Create a new "Limited Administer" permission named "Configure" for managing non-security aspects of a Jenkins instance
+= JEP-223: New "Limited Administer" permission for managing non-security aspects of a Jenkins instance
```
Comment (bitwiseman) [inline jep/223/README.adoc]:
```suggestion
= JEP-223: "Limited Administer" permission for managing non-security aspects of a Jenkins instance
```

File: jep/223/README.adoc [hunk 2]
```diff
@@ -19,7 +19,7 @@ endif::[]
 | 223
 
 | Title
-| Create a new "Limited Administer" permission named "Configure" for managing non-security aspects of a Jenkins instance
+| New "Limited Administer" permission for managing non-security aspects of a Jenkins instance
```
Comment (bitwiseman) [inline jep/223/README.adoc]:
```suggestion
| "Limited Administer" permission for managing non-security aspects of a Jenkins instance
```

**Comment by mikecirioli** [comment]
I am fine with the proposed changes, please merge

**Comment by mikecirioli** [comment]
I am fine with the proposed changes, please merge

**Pull request merged**